lib/cloud_profiler_agent/looper.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'google/apis/errors'
+
+module CloudProfilerAgent
+  # Looper is responsible for the main loop of the agent. It calls a
+  # block repeatedly, handling errors, backing off, and retrying as
+  # appropriate.
+  class Looper
+    def initialize(
+      min_iteration_time: 10,
```

Comment:
The unit would be nice for the times, either as a comment or suffix.

EDIT: I do see the comment on line 19. Still wonder if maybe it'd be nice from the caller's perspective.

Reply:
good point, changed these to `min_iteration_sec`, etc